wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -33,37 +33,139 @@
 #include <wolfssl/internal.h>
 
 #include <stdio.h>
-/* mutex */
-wolfSSL_Mutex tsip_mutex;
-static int tsip_CryptHwMutexInit_ = 0;
+
+
+
 
 /* ./ca-cert.der.sign,  */
 /* expect to have these variables defined at user application */
-extern uint32_t s_flash[];
-extern uint32_t s_inst1[R_TSIP_SINST_WORD_SIZE];
-extern uint32_t s_inst2[R_TSIP_SINST2_WORD_SIZE];
-static const byte *ca_cert_sig;
+extern uint32_t     s_flash[];
+extern uint32_t     s_inst1[R_TSIP_SINST_WORD_SIZE];
+extern uint32_t     s_inst2[R_TSIP_SINST2_WORD_SIZE];
+
 
-/* user key */
+wolfSSL_Mutex       tsip_mutex;
+static int          tsip_CryptHwMutexInit_ = 0;
+static const byte*  ca_cert_sig;
 static tsip_key_data g_user_key_info;
+
 /* tsip only keep one encrypted ca public key */
 #if defined(WOLFSSL_RENESAS_TSIP_TLS)
-static uint32_t g_encrypted_publicCA_key[R_TSIP_SINST_WORD_SIZE];
-static uint32_t g_CAscm_Idx;          /* index of CM table    */
+static uint32_t     g_encrypted_publicCA_key[R_TSIP_SINST_WORD_SIZE];
+static uint32_t     g_CAscm_Idx;          /* index of CM table    */
 #endif
 
-static int tsip_CryptHwMutexInit(wolfSSL_Mutex* mutex) {
+
+
+static int tsip_CryptHwMutexInit(wolfSSL_Mutex* mutex) 
+{
     return wc_InitMutex(mutex);
 }
 
-static int tsip_CryptHwMutexLock(wolfSSL_Mutex* mutex) {
+static int tsip_CryptHwMutexLock(wolfSSL_Mutex* mutex) 
+{
     return wc_LockMutex(mutex);
 }
 
-static int tsip_CryptHwMutexUnLock(wolfSSL_Mutex* mutex) {
+static int tsip_CryptHwMutexUnLock(wolfSSL_Mutex* mutex)
+{
     return wc_UnLockMutex(mutex);
 }
 
+#if defined(WOLFSSL_RENESAS_TSIP_TLS) && (WOLFSSL_RENESAS_TSIP_VER >=109)
+
+static uint32_t GetTsipCipherSuite( 
+                    uint8_t cipherSuiteFirst,
+                    uint8_t cipherSuite)
+{
+    WOLFSSL_MSG(">> GetTsipCipherSuite");
+    uint32_t tsipCipher;
+
+    if(cipherSuiteFirst == CIPHER_BYTE )
+	{
+		switch(cipherSuite){
```

Comment:
Please fix indentation on this line/section, thanks!

Reply:
Several hard tabs should be replaced with spaces.